packages/shared/src/components/cards/brief/BriefCard.tsx

```diff
@@ -0,0 +1,84 @@
+import React, { Children } from 'react';
+import classNames from 'classnames';
+import type { ReactElement, ReactNode } from 'react';
+import { LottieAnimation } from '../../LottieAnimation';
+import {
+  Typography,
+  TypographyColor,
+  TypographyType,
+} from '../../typography/Typography';
+import { ProgressBar } from '../../fields/ProgressBar';
+import { briefCardBg, briefCardBorder } from '../../../styles/custom';
+
+export type BriefCardProps = {
+  className?: string;
+  animationSrc: string;
+  progressPercentage?: number;
+  headnote: ReactNode;
+  title: ReactNode;
+  children?: ReactNode;
```

Comment:
Made it generic with `ReactNode` if for ever reason design wants to customize internals